src/background.ts

```diff
@@ -331,7 +331,9 @@ function getBackupToken(service: string) {
       redirUrl = encodeURIComponent(chrome.identity.getRedirectURL());
       authUrl = `https://login.microsoftonline.com/common/oauth2/v2.0/authorize?client_id=${
         getCredentials().onedrive.client_id
-      }&response_type=code&redirect_uri=${redirUrl}&scope=https%3A%2F%2Fgraph.microsoft.com%2FFiles.ReadWrite.AppFolder%20https%3A%2F%2Fgraph.microsoft.com%2FUser.Read%20offline_access&response_mode=query&prompt=consent`;
+      }&response_type=code&redirect_uri=${redirUrl}&scope=https%3A%2F%2Fgraph.microsoft.com%2FFiles.ReadWrite${
+        localStorage.oneDriveBusiness === "false" ? ".AppFolder" : ""
```

Comment:
Shoud `localStorage.oneDriveBusiness !== "true"` make more sense here?

Reply:
They both work, I just thought that this would be easier to read.

Reply:
My concern is if `localStorage.oneDriveBusiness` is `undefined`, this logic should be incorrect.

Reply:
I'm unfamiliar with extensions or the settings, but I was mostly wondering why there are quotes around the boolean?

Reply:
localStorage can only store string values